jekyll/_components/tabs.md

```diff
@@ -8,30 +8,132 @@ Tabs are used to navigate to related destinations.
+  - Changes with labels (can align to the left, right, or middle): 16px padding for body 20, 8px padding for body 10
+- Tab container height
+  - 48px for body 20
+  - 32px for body 10
+  
+**Light theme**
+- Active tab indicator (tab line)
+  - Blue 50
+- Active tab text label
+  - Blue 60
+- Inactive tab text label
+  - Grey 90 a80
+
+**Dark theme**
+- Active tab indicator (tab line)
+  - Blue 50
+- Active tab text label
+  - Blue 40
+- Inactive tab text label
+  - Grey 10 a80
+
+**Text sizes**
+- Same size as the body text (may be either Body 10 or 20 since there are only two body font sizes)
+  - Body 10 (line height 16) - so it can fit an icon that is 16
+  - Body 20 (line height 24) - so it can fit an icon that is 24
 
 ## States
 
-Describe here the standard states (hover, pressed, disabled) and their according style.
+**Light theme**
+- Hover tab indicator
+  - Grey 90 a10
+- Hover tab overlay
+  - Grey 90 a10
+- Pressed tab indicator
+  - Grey 90 a20
+- Pressed tab overlay
+  - Grey 90 a20
+- Focused box
+  - like button
+
+**Dark theme**
+- Hover tab indicator
+  - Grey 10 a20
+- Hover tab overlay
+  - Grey 10 a10
+- Pressed tab indicator
+  - Grey 10 a20
+- Pressed tab overlay
+  - Grey 10 a20
+- Focused box
+  - like button
 
 ## Behavior
 
-Describe here specific behaviors like scroll, overflow, resizing, and others.
-
+**When there are too many tabs on the right or left, an arrow appears and it becomes scrollable (light theme theme)**
+- Divider
+  - Grey 90 a20
+- Icon fill
+  - Grey 90 a80
+- Gradient
+  - Linear bg-color to transparent
+- Icon hover
+  - Grey 90 a10
+- Icon pressed
+  - Grey 90 a20
+ 
+**When there are too many tabs on the right or left, an arrow appears and it becomes scrollable (dark theme)**
+- Divider
+  - Grey 10 a20
+- Icon fill
+  - Grey 10 a80
+- Gradient
+  - Linear bg-color to transparent
+- Icon hover
+  - Grey 10 a10
+- Icon pressed
+  - Grey 10 a20
+
+**Long labels**
+- Gradient Linear bg-color to transparent
+
+![image](https://user-images.githubusercontent.com/1721875/47156757-10266f00-d2e8-11e8-8ec5-0a36de118fc4.jpg)
+![image](https://user-images.githubusercontent.com/1721875/46953988-a1e76f80-d08f-11e8-9384-2790f8a22afa.jpg)
+![image](https://user-images.githubusercontent.com/1721875/47156799-29c7b680-d2e8-11e8-9ae0-56e18e597b71.jpg)
+
+**The tab bar can be fixed at the top or scroll off the screen**
 
 ## Copy
 
-Describe here content rules.
-
+- Sentence case
+- Short
+- Concise
+- Should be indicative of what is in the tab
 
 ## Accessibility
 
-Describe here accessibility recommendations.
\ No newline at end of file
+There are some useful guidelines on implementing tabs technology-wise: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role, https://inclusive-components.design/tabbed-interfaces/, and https://de.ryerson.ca/wa/aria/jquery/tabpanels.html#?1541694847079#goto_tabsdemo-3. 
+
+What I got from the MDN link:
```

Comment:
perhaps different wording here "what i got" seems casual and out of place.

Reply:
(Could you file that as a followup? 😉)